src/stores/HomeStore.js

```diff
@@ -1,6 +1,6 @@
 import { action, observable } from 'mobx';
-import HOME_ABI from '../abis/HomeBridge.json';
-import BRIDGE_VALIDATORS_ABI from '../abis/BridgeValidators.json'
+import { abi as HOME_ABI } from '../submodules/poa-bridge-contracts/build/contracts/HomeBridgeNativeToErc';
```

Comment:
Is this correct that `submodules/poa-bridge-contracts/build/contracts/HomeBridgeNativeToErc` is specified without `.json`?